draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1933,7 +1969,7 @@ In more detail:
 4.3   attempted_hash = Hash(msg || CTR)
 4.4   h = RS2ECP(attempted_hash)
 4.5   If h is not "INVALID" and cofactor > 1, set h = h * cofactor
-5. Output h
+5. return h
```

Comment:
NOTE: this edit is causing the conflict with master. I suggest removing this edit and then rebasing onto master.

Reply:
Also, ~20 lines above this point is where `G` is used as a point, and probably should be changed to `P`.

Reply:
This is the only remaining issue: in the Related Work appendix, there are 5 uses of the symbol `G` to mean a point that generates a prime-order subgroup. I suggest changing these to `P`.